lib/sorcery/controller/submodules/remember_me.rb

```diff
@@ -36,13 +39,13 @@ def remember_me!
           # Clears the cookie, and depending on the value of remember_me_token_persist_globally, may clear the token value.
           def forget_me!
             current_user.forget_me!
-            cookies.delete(:remember_me_token, domain: Config.cookie_domain)
+            cookies.delete(Config.remember_me_cookie_name, domain: Config.cookie_domain)
```

Comment:
It may make sense to call `.to_sym` here so that users can use a string or symbol in the config, similar to our other config values.